src/api-service/__app__/onefuzzlib/job_templates/defaults/libfuzzer.py

```diff
@@ -330,6 +330,17 @@
                 ),
             ],
         ),
+        UserField(
```

Comment:
I don't think this is what you meant.

This modifies the first task in the libfuzzer template list, which is libfuzzer_fuzz, and adds the value `report_list`.  I thought `report_list` was only used during the new regression task.

Reply:
I think this will likely need to be it's on JobTemplate